src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -42,6 +42,9 @@ public static class LanguageParser
                                                                                                           (?<slovak>\bSK\b)",
                                                                 RegexOptions.Compiled | RegexOptions.IgnorePatternWhitespace);
 
+        private static readonly Regex GermanDualLanguageRegex = new (@"(?<!web-)\bdl\b", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
There are also many cases where we see WEB.DL and not just WEB-DL. We need to account for those as well. 

```
Shark.Bait.2022.1080p.WEB.DL.DD5.1.H.x264-EVO
Sherni.2021.2160p.AMZN.WEB.DL.DDP5.1.HEVC-TELLY
```

Reply:
Right, I updated it to `(?<!WEB[-_. ]?)\bDL\b`, similar to how it is in the QualityParser
https://github.com/Radarr/Radarr/blob/122d0056eabf2e7c1dc7fc1d210cdd057ddadb19/src/NzbDrone.Core/Parser/QualityParser.cs#L19